src/main/java/jenkins/plugins/nodejs/tools/NpmPackagesBuildWrapper.java

```diff
@@ -1,112 +1,50 @@
 package jenkins.plugins.nodejs.tools;
 
-import com.google.common.base.Throwables;
 import hudson.*;
-import hudson.model.AbstractBuild;
 import hudson.model.AbstractProject;
-import hudson.model.BuildListener;
 import hudson.model.Computer;
 import hudson.model.Run;
+import hudson.model.TaskListener;
 import jenkins.plugins.nodejs.NodeJSPlugin;
-import hudson.tasks.BuildWrapper;
 import hudson.tasks.BuildWrapperDescriptor;
+import jenkins.tasks.SimpleBuildWrapper;
 import org.kohsuke.stapler.DataBoundConstructor;
 
-import java.io.File;
 import java.io.IOException;
 
 /**
  * @author fcamblor
  */
-public class NpmPackagesBuildWrapper extends BuildWrapper {
+public class NpmPackagesBuildWrapper extends SimpleBuildWrapper {
 
-    private String nodeJSInstallationName;
+    private final String nodeJSInstallationName;
 
     @DataBoundConstructor
     public NpmPackagesBuildWrapper(String nodeJSInstallationName){
```

Comment:
BTW it seems there is no autofill for this, which is a pity. Not an issue in your PR but something you might want to fix at the same time, or in parallel.